readme.md

```diff
@@ -164,7 +164,7 @@ lighthouse -GA=./gmailartifacts https://gmail.com
 
 #### Notes on Error Reporting
 
-The first time you run the CLI you will be prompted with a message asking you if Lighthouse can anonymously report runtime exceptions. The Lighthouse team uses this information to detect new bugs and avoid regressions. Opting out will not affect your ability to use Lighthouse in any way. [Learn more](https://github.com/GoogleChrome/lighthouse/blob/master/docs/error-reporting.md).
+The first time you run the CLI, you will be prompted with a message asking you if Lighthouse can anonymously report runtime exceptions. The Lighthouse team uses this information to detect new bugs and avoid regressions. Opting out will not affect your ability to use Lighthouse in any way. [Learn more](https://github.com/GoogleChrome/lighthouse/blob/master/docs/error-reporting.md).
```

Comment:
I disagree with this.  I think it is fine without the comma, the beginning isn't really a distinct introductory phrase that would warrant a comma.  I don't think :man_shrugging:

If the first part was reworded to be more distinct I could see it.